telco-hub/configuration/README.md

```diff
@@ -0,0 +1,218 @@
+* Configured and existing Openshift CatalogSources for `redhat-operators-disconnected` and `certified-operators-disconnected`.
+
+## Init phase (prepare ArgoCD)
+
+ArgoCD is one of the main key components of the Telco Hub. At the same time, we can deploy the Telco Hub using ArgoCD (recommended procedure). Therefore, to have a Telco Hub with ArgoCD, first, we have to have ArgoCD to create the Telco Hub. This is the init phase.
+
+In this init phase we will install ArgoCD with the existing `reference-crs` for gitops. But, you could provide ArgoCD differently, or maybe, you already have ArgoCD in your cluster. 
+
+In case you want to proceed with the existing `reference-crs` for gitops (recommended): 
+
+```bash
+oc apply -f reference-crs/required/gitops/clusterrole.yaml \
+  -f reference-crs/required/gitops/clusterrolebinding.yaml \
+  -f reference-crs/required/gitops/gitopsNS.yaml \
+  -f reference-crs/required/gitops/gitopsOperatorGroup.yaml \
+  -f reference-crs/required/gitops/gitopsSubscription.yaml
+```
+
+Wait the operator to be installed:
+
+```bash
+> oc -n openshift-gitops-operator get subscriptions.operators.coreos.com openshift-gitops-operator -o jsonpath='{.status.state}'
+
+AtLatestKnown
+
+> oc -n openshift-gitops-operator get pod
+NAME                                                         READY   STATUS    RESTARTS   AGE
+openshift-gitops-operator-controller-manager-d97fddc-9zmrn   2/2     Running   0          21m
+
+> oc -n openshift-gitops get pod
+NAME                                                          READY   STATUS    RESTARTS   AGE
+cluster-7b65f74f8f-sbx24                                      1/1     Running   0          37s
+gitops-plugin-7d8b6d777b-5npgj                                1/1     Running   0          37s
+kam-7bc6f69fcd-jrtgv                                          1/1     Running   0          37s
+openshift-gitops-application-controller-0                     1/1     Running   0          35s
+openshift-gitops-applicationset-controller-5cddb476fc-q5shw   1/1     Running   0          35s
+openshift-gitops-dex-server-954f978c9-2lp44                   1/1     Running   0          35s
+openshift-gitops-redis-7ff87f9b48-8ld9x                       1/1     Running   0          35s
+openshift-gitops-repo-server-6ccffb9695-pc8bj                 1/1     Running   0          35s
+openshift-gitops-server-845d6798-9c5tv                        1/1     Running   0          35s
+```
+
+## Tune your own overlay layer
+
+Before creating the Telco Hub ArgoCD Application, you have to select the different optional component, and configure all of them. This is done using the different available kustomize patches. 
+
+At this point, in general, you will fork this repo to tune the different kustomize patches and to select the optional components. There exists a `kustomize.yaml` with all the information:
```

Comment:
They should do their overlay work before the first apply of argocd/application above.